core/domain/question_domain.py

```diff
@@ -432,20 +425,19 @@ def validate(self):
             ValidationError: One or more attributes of question summary are
                 invalid.
         """
-        err_dict = html_validation_service.validate_rte_format(
-            [self.question_content], feconf.RTE_FORMAT_CKEDITOR)
-        for key in err_dict:
-            if err_dict[key]:
-                raise utils.ValidationError(
-                    'Invalid html: %s for rte with invalid tags and '
-                    'strings: %s' % (self.question_content, err_dict))
-
-        err_dict = html_validation_service.validate_customization_args([
-            self.question_content])
-        if err_dict:
+        if not isinstance(self.id, basestring):
+            raise utils.ValidationError(
+                'Expected id to be a string, received %s' % self.id)
+
+        if not isinstance(self.creator_id, basestring):
```

Comment:
Thanks for adding validation for ids. 
Do we also have to validate the `last_updated_msec` and `created_on_msec`?

Reply:
Done. Also filed #7146!